api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -461,7 +461,7 @@ public void saveGlobalProperties_shouldAssignUuidToAllNewProperties() throws Exc
 	@Verifies(value = "should return all global properties in the database", method = "getAllGlobalProperties()")
 	public void getAllGlobalProperties_shouldReturnAllGlobalPropertiesInTheDatabase() throws Exception {
 		executeDataSet(ADMIN_INITIAL_DATA_XML);
-		Assert.assertEquals(19, Context.getAdministrationService().getAllGlobalProperties().size());
+		Assert.assertEquals(20, Context.getAdministrationService().getAllGlobalProperties().size());
```

Comment:
It is good to add such test data to new test data file instead of the existing that you avoid touching these existing tests